### PR TITLE
Create openingHours modal, controller and postman test

### LINKED ISSUE #13: Create Opening Hours Modal
As a user I want to be able to see what hours the hair salon is open so I know when I can go there for drop in, even for holidays.

- [x] There shall be a modal for opening hours
- [x]  The modal shall have Name(String), Description(String), active(boolean) and opening hours(string)

### LINKED ISSUE #14: Create Opening Hours Controller
As a owner/employee I want to be able to change our opening hours so that people know when we are open.

- [x] There shall be a opening hours controller
- [x] It shall allow for the GET, POST, PUT and DELETE to the opening hours items

### LINKED ISSUE #15: Postman tests Opening Hours
The repository shall contain Postman CRUD tests that is repeatable(the delete targets the one posted).

- [x] post
- [x] get
- [x] get by id
- [x] put
- [x] delete
- [x] repeatable

### PR DESCRIPTION
**Pull request by harring**
Created openingHours model which fixes #13 , controller which fixes #14 and added postman tests which fixes #15 .

### CODE REVIEW

File: backend/models/openingHours.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+var mongoose = require('mongoose');
+var Schema = mongoose.Schema;
+
+var openingHoursSchema = new Schema({
```
Comment (Toerngren) [inline backend/models/openingHours.js:4]: Should we add "required: true" as an argument on each attribute?

**Review by Toerngren — COMMENTED** [review]
Would it be easier to update the postman file after each addition (of tests) or add them separately?

**Comment by Toerngren** [comment]
Hello @harring, just some thoughts to discuss. Overall, code looks good!

**Comment by harring** [comment]
> 

If we keep all ready/finished tests in one folder on Postman we can just export that when it needs updating. Should not be that often.

File: backend/models/openingHours.js (hunk 1 above)
Comment (harring) [reply backend/models/openingHours.js:4]: Yes, we should. I will push an update.

**Comment by harring** [comment]
Code has been updated:

- All attributes are now required.

**Review by Toerngren — APPROVED** [review]
Nice. Good, I agree about error handling and will add that later. Good to add an issue for that.

**Pull request merged**